packages/appsync-modelgen-plugin/src/__tests__/visitors/__snapshots__/appsync-dart-visitor.test.ts.snap

```diff
@@ -2853,7 +2853,7 @@ class Blog extends Model {
   Map<String, dynamic> toJson() => {
         'id': id,
         'name': name,
-        'posts': posts?.map((e) => e?.toJson())?.toList(),
+        'posts': posts?.map((Post? e) => e?.toJson()).toList(),
```

Comment:
Can `Post` be null?

Reply:
Post is not nullable when in list, which I think is a bug to fix. More details in my comment below